STAGES.md

```diff
@@ -45,9 +47,8 @@ tied to any particular solution.
 
 > “This idea is not silly.”
 
-A **Working Draft** [championed] by a [W3C Working Group]. It should be
-considered relatively unstable and subject to change. Stage 2 features are tied
-to a particular way of solving a problem.
+A **Working Draft** [championed] by a [W3C Working Group] or implemented by at least two [recognized browser vendors]. It should be considered relatively unstable and subject to change. Stage 2 features are tied to a particular way of solving a problem. This feature should be considered somewhat unstable and subject to change.
```

Comment:
```suggestion
A **Working Draft** [championed] by a [W3C Working Group] or implemented by at least two [recognized browser vendors]. It should be considered relatively stable and but still subject to change. Stage 2 features are tied to a particular way of solving a problem.
```

When implemented by two vendors it is much less likely to change drastically. It is very rare for two vendors to implement a feature at the same time without communicating and working in the same direction.

Reply:
I agree with your comment but my reasoning was that it can be spec status or browser implementation. Thoughts on that?

Reply:
I missed the `or` here. :)

I think there has to be a stage that is set once and only if it is implemented in at least two browsers, regardless of spec status.

This stage is the most interesting one to use.
You get to use modern features with the safety of knowing they are unlikely to change. Without having to wait for the one vendor that is late with its implementation.

Can we redefine stage 2 to fit this description?
I personally would never use stage 2 as it is currently defined because I would not know which of the two conditions is true.

Alternatively we could add an extra stage solely for "implemented by 2 vendors".

Thoughts?

Reply:
I'm a bit torn. I _think_ by the time it has reached Working Draft it's likely already implemented. If you look at env-function for example it's still 0 but this would bump it to 3 given the surprisingly good support.

Reply:
I think the spec status is given too much weight in the current definition.
Currently it is possible for a feature to be stage 4 without any browser implementation at all purely based on the it being a `Recommendation`. Whereas actual implementations are a much better indicator for stability of a feature.

>  If you look at env-function for example it's still 0 but this would bump it to 3 given the surprisingly good support.

It is the inverse case that I am worried about :)
Any (sub) feature without implementations is highly likely to change.

A good example of this the gamut mapping in `lab`/`lch`/`color`.
This is a recent addition to a spec that is a Working Draft.
But it doesn't have any implementations so should be `stage 0`.

Maybe I am trying to solve something in the wrong place?

Reply:
discussed with @Antonio-Laguna and this has been resolved.
We decided to keep `stage` as it currently stands in this PR and add separate info for the implementation stats.